app/code/Magento/Customer/Controller/Account/CreatePassword.php

```diff
@@ -42,25 +41,34 @@ class CreatePassword extends \Magento\Customer\Controller\AbstractAccount implem
      */
     private $confirmByToken;
 
+    /**
+     * @var \Magento\Customer\Model\ForgotPasswordToken\GetCustomerByToken
```

Comment:
Looks like this class already imported, so we can change it to something like this
```suggestion
     * @var GetCustomerByToken
```